server/fast_forward.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/google/go-github/v39/github"
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+const (
+	cloudBranchName = "heads/cloud"
+	mainBranchName  = "heads/master"
+	processRepo     = "mattermost-server"
+)
+
+// HasCloudFF is true if body contains "/cloud-ff"
+func (e *issueCommentEvent) HasCloudFF() bool {
+	return strings.Contains(strings.TrimSpace(e.Comment.GetBody()), "/cloud-ff")
+}
+
+func (s *Server) performFastForwardProcess(ctx context.Context, issue *model.Issue, comment, user string) error {
+	if issue.State == model.StateClosed || issue.RepoName != processRepo { // we will perform this only on mm-server issues
+		return nil
+	}
+
+	// Don't start process if the user is not a core committer
+	if !s.IsOrgMember(user) {
+		return nil
+	}
+
+	// Check if the args are correct
+	command := getFFCommand(comment)
+	args := strings.Split(command, " ")
```

Comment:
Can we not do the arg generation automatically based on the current date? I'd prefer it to be as hassle-free as possible.